### ISSUE TITLE
Output igual al esperado pero arroja error igual

### ISSUE THREAD

**Issue by IgnacioMejias** [RodrigoToroIcarte/IIC2113-2023-1#177]
Hola! estaba probando el test 1 de 05-SuperstarAbilities, y tengo el problema de que arrojo exactamente lo esperado, pero me dice que no lo es. Adjunto imagenes. 

esto espera el test: 

![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/89417528/4bfbb375-5e9c-4c22-9c20-7f1ca4d8f1a9)

esto arroja mi codigo: 

![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/89417528/46e636f8-89fb-454a-930c-17440eb60e93)

No entiendo que puede estar pasando. Ademas, probándolo manualmente y revisando el archivo de test esperado coincide todo. No estoy usando nada estático, lo único "raro" que tengo es que en mi clase Superstar instancio view así: 

protected View _view = View.BuildConsoleView();

**Comment by IgnacioMejias** [comment]
ya lo solucioné, el problema es que al crear una nueva vista en el superstar fallan los casos. Lo solucione pasándole directamente la vista de Game, por si a alguien más le pasa.

**Comment by RodrigoToroIcarte** [comment]
Efectivamente. En todo el código debes usar la misma `view` que se le entrega al `Game` porque los test cases solo validan el input/output generado desde esa view.

**Comment by Crisax234** [comment]
A mi me ocurre ahora que me pasa el mismo error y he utilizado la misma instancia de _view que me dan en game, que puede ser?
![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/88630837/f946e574-1f53-4490-9db3-d79ce281e615)

**Comment by RodrigoToroIcarte** [comment]
Necesitaría más info para saber qué está pasando. Pero creo que te está reclamando que esperaba ver el `"0- [MANEUVER]..."` y vio el `"--------------------"`. Y lo otro que hay distinto es que el output esperado es  `MANEUVER` en mayúscula y tu output está con minúscula.

¿el `"--------------------"` es de un writeline tuyo?
